src/snippets/shield/quick-start/nodejs/Step1.mdx

```diff
@@ -0,0 +1,7 @@
+import SelectableContent from "@/components/SelectableContent";
+
+<SelectableContent client:idle syncKey="packageManager">
+  <Fragment slot="npm">```sh npm i @arcjet/node ```</Fragment>
```

Comment:
The code fence should be on a new line for each of these otherwise it doesn't render correctly. See the example in `src/snippets/shield/quick-start/nextjs/Step1.mdx`

```suggestion
  <Fragment slot="npm">
  ```sh 
  npm i @arcjet/node 
  ```
  </Fragment>
```

You may need to add `{/* prettier-ignore */}` to prevent them being auto formatted.

Reply:
Done!